WalletWasabi/WebClients/Wasabi/IHttpClientFactory.cs

```diff
@@ -3,8 +3,26 @@
 
 namespace WalletWasabi.WebClients.Wasabi
 {
-	public interface IBackendHttpClientFactory
```

Comment:
Rename the file to `IWasabiHttpClientFactory.cs`?

Reply:
Yes. Thanks.